src/python/turicreate/test/test_sframe_s3.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2020 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+from __future__ import print_function as _  # noqa
```

Comment:
I don't understand the `noqa` comment on this and the next two lines.

Reply:
pass flake8 and pylint.